src/main/java/org/openstreetmap/atlas/geography/atlas/change/AtlasChangeGenerator.java

```diff
@@ -76,8 +76,13 @@
                     newBounds = newBounds.combine(bounds);
                 }
             }
+            CompleteNode newCompleteNode = originalCompleteNode.withBoundsExtendedBy(newBounds);
+            if (originalCompleteNode.getLocation() == null && originalNode != null)
+            {
+                newCompleteNode = newCompleteNode.withLocation(originalNode.getLocation());
```

Comment:
Not sure of the context, but is the originalNode guaranteed to have a location?

Reply:
Yes, because it comes directly from an `Atlas`. See line 63 of this file.